Rovers/Rovers.tex

```diff
@@ -0,0 +1,387 @@
+    \hline
+    Rocker-bogie-like suspension & This is the NASA preferred rover suspension style. \\
+    Ability to control remotely (such as a gamepad) & Remote control is essential for rovers. \\
+    Ability to traverse terrain such as gravel, cement, and asphalt & Multi-terrain traversal is important for rovers. \\
+    Technical report on GitHub & This will allow for future teams to easily learn from this teams experience. \\
+    Rover hardware & Having a physical rover is important for a rover project as it allows for lessons to be learned in construction. \\ 
+    Present at Imagine RIT & This is essential for promotional value. \\
+    Self-driving & Partially autonomous navigation for future URC readiness. \\
+    \hline
+    \end{tabularx}
+    }
+\label{tab:mvp-one}
+\end{table}
+
+\subsection{Improvements}
+\label{subsec:improvements}
+The MVP is intended to be flexible in implementation depending on funding. 
+The purpose of the project is to gain experience in rovers and robotics and this can be done in a number of form factors.
+The size and materials are not defined because of this. 
+Should the funding allow for it there are many improvements that can be made beyond the MVP. 
+This allows for more experience to be gained, thus fulfilling the goals of the project.
+
+\begin{table}[ht!]
+    \caption{Improvements}
+    \centering
+    {\renewcommand{\arraystretch}{1.5}
+    \begin{tabularx}{\linewidth}{LL}
+    \hline
+    \textbf{Feature} & \textbf{Reason} \\  
+    \hline
+    Lidar based CV & Autonomous navigation for future URC readiness. \\
+    Improved navigation with use of GPS system & Additional navigation options. \\
+    Robotic arm with 3+ degrees of freedom & URC readiness and rover capabilities. \\ 
+    Drill or grasper attachment & Increased capabilities of the robotic arm. \\
+    Soil sample analysis with Archimedes' screw & Scientific possibilities. \\
+    Larger and or faster rover & Able to traverse obstacles of greater magnitude or hold more scientific equipment for future expansion. \\
+    Solar cells & Additional long-term navigation viability and experience. \\
+    Web control interface with GUI & Easier navigation. \\
+    Durable construction & Carbon fiber or similar materials for long term viability. \\
+    On board batteries & Lithium Ion, 18650s or similar battery technology for remote abilities. \\
+    Long range communications & Ability to communicate with the rover at lengths of up to one kilometer allows for URC readiness. \\
+    \hline
+    \end{tabularx}
+    }
+\label{tab:mvp-two}
+\end{table}
+
+\section{Secondary Objectives}
+\label{sec:secondary-obj}
+The study will also investigate the University Rover Competition (URC) as hosted by the Mars Society as a potential long term goal. 
+The team will accomplish this by evaluating how many of the MVP improvements are accomplished. 
+If none of these improvements are not able to be implemented then the team is not near URC ready.
```

Comment:
There's an extra negation in this sentence.

Reply:
Done